data21/focuses.json

```diff
@@ -128,6 +128,16 @@
         "name": "Mobile and GUI Engineering",
         "url": "allModules/28254_M_MGE.json"
       },
+      {
```

Comment:
Is GenAI really part of Frontend Engineering in Studienreglement 21? Is there another place we can get this information from, to cross check?

Reply:
oh good catch. It's not supposed to be. I'll remove it manually.